content/documentation/pages/5-feature-guides/2-batch/3-partitioning.md

```diff
@@ -1,7 +1,260 @@
 ---
 path: 'feature-guides/batch/partitioning/'
-title: 'Data Partitioning'
-description: 'How to use partitioning'
+title: 'Remote Batch Partitioning'
+description: 'How to use remote batch partitioning'
 ---
 
-Introduction to partitioning
+# Introduction to remote partitioning
+
+The Batch Developer Guide showed you how to create the typical single threaded Spring Batch application.
+While this is a great solution for most batch applications, there are cases where a particular step may take a significant amount time to do the work required.
+Spring Batch has a solution that allows a [Batch Job](https://docs.spring.io/spring-batch/4.0.x/reference/html/domain.html#job) to partition a [Step](https://docs.spring.io/spring-batch/4.0.x/reference/html/domain.html#step) execution, where each partition handles a segment of work.
+In short partitioning allows multiple versions of large batch applications to run concurrently.
+The purpose of this is to reduce the elapsed time required to process long batch jobs.
+Processes that can be successfully partitioned are those where the input file can be split and/or the main database tables partitioned to allow the application to run against different sets of data.
+
+So lets say we have a 3 Step Job:
+
+![Batch Partitioning](images/batch-partition.png)
+
+The Job runs on the left-hand side as a sequence of Step instances, and `Step 2` is labeled as `<Manager>`.
+The `Manager step` is responsible for allocating the work to and launching each `worker`.
+In this case the `worker` is a another instance of the Spring Batch application that is executed with a specific profile enabled.
+The workers in our diagram are actually copies of our Spring Batch application that are deployed to the platform.
+The messages sent by the manager to the workers in this pattern do not need to be durable or have guaranteed delivery.
```

Comment:
We are saying something about "messages", but we haven't described what the messages are in this context.